components/layermanager/layermanager.service.ts

```diff
@@ -640,7 +640,7 @@ export class HsLayerManagerService {
   removeAllLayers(): void {
     const to_be_removed = [];
     this.HsMapService.map.getLayers().forEach((lyr) => {
-      if (lyr.get('removable') == undefined || lyr.get('removable') == true) {
+      if (lyr.get('removable') == true) {
```

Comment:
Could you add a comment so we dont forget how its supposed to work?

Reply:
Yes, of course